span.go

```diff
@@ -225,18 +234,66 @@ func (s *Span) OperationName() string {
 	return s.operationName
 }
 
+// Retain increases object counter to increase the lifetime of the object
+func (s *Span) Retain(count ...int32) *Span {
```

Comment:
is the vararg argument really necessary?

Reply:
No, It was needed for the first commit implementation.
Thank you, I will remove it